charts/seed-bootstrap/charts/nginx-ingress/templates/vpa.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: "autoscaling.k8s.io/v1beta2"
+kind: VerticalPodAutoscaler
```

Comment:
In gardener we are using the hvpa in different ways.
Option 1) Disabling the hpa component and just use the vpa part of the hvpa as done for [Loki](https://github.com/gardener/gardener/blob/69cc831f7c7fdc91405e16fe792a2f5b433f23c6/charts/seed-bootstrap/charts/loki/templates/hvpa.yaml#L17).
-> I don't think we are going to see heavy resource spikes for the nginx-ingress-controller (which was meant to be better solved with the hvpa vpa in comparison to normal vpa). Especially as we have 3 replicas the normal vpa should be able to handle the scaling.
Option 2) Also use the hpa component of the hvpa.
-> I do not see us benefiting from hpa here. 3 replicas will be fine and we probably do not need more. We also do not want less to prevent downtimes. So just vanilla 3 replicas without hpa seems to be sufficient. And as it is sufficient I prefer the solution with less complexity.

Reply:
thanks for the explanation!